ideascube/configuration/errors.py

```diff
@@ -0,0 +1,21 @@
+class InvalidConfigurationValue(Exception):
```

Comment:
errors.py is a functional name.
exceptions.py is more a implementation detail name.
But both are valid, I don't especially care.

However, I prefer to name exception themselves with "Error" in the name.
Python itself do the same, all exception for error contain Error if the name.
Other exceptions are not Error (StopIteration, KeyboardInterupt, GeneratorExit, ...)